assets/constants.ts

```diff
@@ -5607,6 +5612,10 @@ export default {
   // A regular expression for allowed characters in URL fragment fields.
   "VALID_URL_FRAGMENT_REGEX": "^[a-z]+(-[a-z]+)*$",
 
+
+  // A regular expression for allowed characters in URL fragment fields.
+  "VALID_BLOG_POST_TITLE_REGEX": "^[a-zA-Z][a-zA-Z ]+(-[a-zA-Z]+)*$",
```

Comment:
```suggestion
  "VALID_BLOG_POST_TITLE_REGEX": "^[a-zA-Z][a-zA-Z]+(-[a-zA-Z]+)*$",
```

Reply:
The space [a-zA-Z ] is added to allow space between words.
This check will ensure that title does not start with space but allows it between words after.

Reply:
In that case shouldn't you use `\s`?

Reply:
We provide the regex expression in raw form, I think. I tried writing [a-zA-Z/s] it gives the following error:
![Screenshot from 2021-06-20 12-42-56](https://user-images.githubusercontent.com/68547101/122672525-88f0ce00-d1e9-11eb-9bc1-17bc1739c794.png)

Reply:
@seanlip I thought we were forbidding spaces and non-standard characters from ID, should we enforce that here or add a TODO?

Reply:
It's for the blog post title, not ID. The title will be something:- 'Help Provide Quality Education through Oppia'

Reply:
Got it, sorry for confusion. LGTM in that case.